pkg/framework/context/context.go

```diff
@@ -83,9 +84,14 @@ func (p *Provider) Packager() envelope.Packager {
 	return p.packager
```

Comment:
now we have Packager(), Packers() and DefaultPackager().. back to my first comment above.. Packager interface becomes redundant

Reply:
I agree that the interface is redundant - with packing algorithm configuration available through `WithDefaultPacker()` and `WithPackers()`, we don't need to allow the `Packager` layer to be injectable.

Reply:
I've removed the injection.